src/modules/Security.rb

```diff
@@ -291,46 +293,19 @@ def ReadServiceSettings
       nil
     end
 
+    def inittab_shutdown_configured?
+      inittab = SCR.Dir(path(".etc.inittab"))
+      inittab.include? "ca"
```

Comment:
Hehe, ok, i will put in this case, but in the guide linked for example you have puts 'whatever' that is also a method without parenthesis :P with just one parameter have them or not is not really a problem but i will try to do as proposed in advance